### PR TITLE
Initializing the repository

### PR DESCRIPTION
**Pull request by cpinjani**
This PR covers:
- Cypress config setup
- Installing Rancher
- Run cypress tests for performing rancher bootstrap and navigate to fleet menu

Example run on custom self-hosted runner:
https://github.com/cpinjani/fleet-e2e/actions/runs/7695014098

### CODE REVIEW

File: tests/cypress/latest/e2e/unit_tests/first_connection.spec.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+limitations under the License.
+*/
+
+import * as cypressLib from '@rancher-ecp-qa/cypress-library';
+import { qase } from 'cypress-qase-reporter/dist/mocha';
+
+
+Cypress.config();
+describe('First login on Rancher', () => {
+  qase(46,
```
Comment (sbulage) [inline tests/cypress/latest/e2e/unit_tests/first_connection.spec.ts:21]: @cpinjani this QASE is not present in Fleet project, can you please explain from where it is located?

File: tests/cypress/latest/e2e/unit_tests/menu.spec.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Cypress.config();
+describe('Menu testing', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit('/');
+    cypressLib.burgerMenuToggle();
+  });
+
+  qase(3,
```
Comment (sbulage) [inline tests/cypress/latest/e2e/unit_tests/menu.spec.ts:27]: Same for this

File: tests/cypress/latest/e2e/unit_tests/user.spec.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Cypress.config();
+describe('User role testing', () => {
+  const stdUser       = "std-user"
+  const uiPassword    = "rancherpassword"
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  qase(15,
```
Comment (sbulage) [inline tests/cypress/latest/e2e/unit_tests/user.spec.ts:28]: And this too.

**Review by sbulage — CHANGES_REQUESTED** [review]

File: tests/cypress/latest/e2e/unit_tests/first_connection.spec.ts (hunk 1 above)
Comment (cpinjani) [reply tests/cypress/latest/e2e/unit_tests/first_connection.spec.ts:21]: Good point, its a placeholder and doesn't corresponds to QASE number as of now.
Can be changed it to matching one (post QASE integration with automation runs)

Comment (sbulage) [reply tests/cypress/latest/e2e/unit_tests/first_connection.spec.ts:21]: For pose QASE integration, do we need to add those test cases in the Fleet project ?

Comment (cpinjani) [reply tests/cypress/latest/e2e/unit_tests/first_connection.spec.ts:21]: They can be added if missing

Comment (sbulage) [reply tests/cypress/latest/e2e/unit_tests/first_connection.spec.ts:21]: Okay, thanks @cpinjani :smile:

**Review by mmartin24 — APPROVED** [review]
LGTM. 
Thanks a lot!

**Review by sbulage — APPROVED** [review]

**Comment by thehejik** [comment]
Apologize, accidentally merged, I somehow bypassed `main` branch protection.

**Pull request merged**